lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -305,9 +310,9 @@ public function formatDateTime(
         }
 
         $formatter = $this->dateFormatterFactory->create(
-            (string)($locale ?: $this->_localeResolver->getLocale()),
-            (int)($dateType ?? \IntlDateFormatter::SHORT),
-            (int)($timeType ?? \IntlDateFormatter::SHORT),
+            (string) $locale ?: $this->_localeResolver->getLocale(),
+            (int) $dateType ?: \IntlDateFormatter::SHORT,
+            (int) $timeType ?: \IntlDateFormatter::SHORT,
```

Comment:
```suggestion
            (int) $dateType ?? \IntlDateFormatter::SHORT,
            (int) $timeType ?? \IntlDateFormatter::SHORT,
```

Reply:
returned it as it was before and add annotation to ignore phpstan instead